README.md

```diff
@@ -36,10 +50,15 @@ null == undefined
 // => true
 ```
 
-This is confusing and bad! It makes no sense that the string `'0'` is equal to the boolean `false` or that `null` and `undefined` — two **completely different** primitive data types — are equivalent. ***Never use `==` for comparisons***.
+This is confusing and inaccurate! It makes no sense that the string `'0'` is
+equal to the boolean `false` or that `null` and `undefined` — two **completely
+different** data types — are equivalent. ***Never use `==` for comparisons***.
```

Comment:
Again, I wonder if this will always be the case. I do strict operators by default, but I'm wondering if there's ever edge-cases where this would be the exception.

Reply:
I think my gut instinct at this level is to teach them this is the absolute rule, and then at some point they'll figure out whatever the edge cases are. I'm trying to figure out what they are and I can't come up with one off the top of my head.